aten/src/ATen/CMakeLists.txt

```diff
@@ -250,6 +256,14 @@ if(USE_XPU)
   list(APPEND ATen_XPU_INCLUDE ${PROJECT_SOURCE_DIR}/third_party/ideep/mkl-dnn/include)
   list(APPEND ATen_XPU_INCLUDE ${XPU_MKLDNN_INCLUDE})
 
+  if(USE_KIENTO)
+    find_package(Pti 0.7.0 QUIET)
```

Comment:
let's remove the version check here.
If need version check, we should check in code.